cityofaustin/atd-moped
# Add phase dates and personnel roles columns to Project List View

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/9073
https://github.com/cityofaustin/atd-data-tech/issues/9078

Big Thank you to @frankhereford for his help refining the View

## Testing
**URL to test:** 

Run locally or use moped-test

**Steps to test:**
Add a construction phase to a project, with a start date. 
Add a completion phase to a project, with an end date. 

Check to see that both appear on the projects list page. 
Advanced search using those fields. Note, depending on when you are adding dates to the phases, the display date may not match the database date (https://github.com/cityofaustin/atd-data-tech/issues/5682). 

To test the personnel, add Inspectors and Designers to a project. See the records on the project list page. Add more than one user, delete some from the personnel table to test that soft deleted records do not show up on the page. 

---
#### Ship list
